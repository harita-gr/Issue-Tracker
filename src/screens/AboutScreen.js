import React from 'react'
import { Container,Image } from 'react-bootstrap'

function AboutScreen() {
    return (
        <Container className='my-5'>
            <h3>About: This application is used to track status of raised issues</h3>
            <Image className='my-3 mx-auto aboutImage' src="https://fiverr-res.cloudinary.com/images/t_main1,q_auto,f_auto,q_auto,f_auto/gigs/101240985/original/0e4c8581b29c00ab6ae3711ac0726a5f7345e055/find-defects-on-your-website.jpg" />
        </Container>
    )
}

export default AboutScreen
