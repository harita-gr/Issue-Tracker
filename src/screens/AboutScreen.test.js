import React from 'react';
import {shallow } from 'enzyme';
import AboutScreen from './AboutScreen'

describe('<AboutScreen/>', () => {

    let wrapper;

    beforeEach(() =>{
        wrapper = shallow(<AboutScreen/>); 
     });

     
        it('should render correctly using Full DOM', () => {
            expect(wrapper).toMatchSnapshot();
        });

        it('render the correct heading name',() => {
            expect(wrapper.find("h3").text()).toContain("About")
        })

});