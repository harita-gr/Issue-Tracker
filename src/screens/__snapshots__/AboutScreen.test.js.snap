// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutScreen/> should render correctly using Full DOM 1`] = `
<Container
  className="my-5"
  fluid={false}
>
  <h3>
    About: This application is used to track status of raised issues
  </h3>
  <Image
    className="my-3 mx-auto aboutImage"
    fluid={false}
    rounded={false}
    roundedCircle={false}
    src="https://fiverr-res.cloudinary.com/images/t_main1,q_auto,f_auto,q_auto,f_auto/gigs/101240985/original/0e4c8581b29c00ab6ae3711ac0726a5f7345e055/find-defects-on-your-website.jpg"
    thumbnail={false}
  />
</Container>
`;
